scripts/deploy/deploy-webapi.ps1

```diff
@@ -42,22 +42,55 @@ if ($LASTEXITCODE -ne 0) {
 }
 
 Write-Host "Getting Azure WebApp resource name..."
-$webappName=$(az deployment group show --name $DeploymentName --resource-group $ResourceGroupName --output json | ConvertFrom-Json).properties.outputs.webapiName.value
-if ($null -eq $webAppName) {
+$deployment=$(az deployment group show --name $DeploymentName --resource-group $ResourceGroupName --output json | ConvertFrom-Json)
+$webApiUrl = $deployment.properties.outputs.webapiUrl.value
+$webApiName = $deployment.properties.outputs.webapiName.value
+
+if ($null -eq $webApiName) {
     Write-Error "Could not get Azure WebApp resource name from deployment output."
     exit 1
 }
 
-Write-Host "Azure WebApp name: $webappName"
+Write-Host "Azure WebApp name: $webApiName"
 
 Write-Host "Configuring Azure WebApp to run from package..."
-az webapp config appsettings set --resource-group $ResourceGroupName --name $webappName --settings WEBSITE_RUN_FROM_PACKAGE="1" | out-null
+az webapp config appsettings set --resource-group $ResourceGroupName --name $webApiName --settings WEBSITE_RUN_FROM_PACKAGE="1" | out-null
```

Comment:
Please add `-o none` to this command so that it does not output anything. This command has the potential to leaks secrets otherwise.

Reply:
If you see this command anywhere else, please add it there also.

Reply:
OK.  Replacing the redirections of output to null with -o none